client/components/UploadPopUp.jsx

```diff
@@ -0,0 +1,87 @@
+import { gql, useMutation } from "@apollo/client";
+import React, { useEffect, useRef, useState, setState } from "react";
+import { ImExit } from "react-icons/im";
+import {axios} from 'axios';
```

Comment:
Why do we need axios?

Reply:
I installed it because I needed it for the first way I attempted the file upload